cpp/src/IceStorm/Election.ice

```diff
@@ -37,23 +37,24 @@ module IceStormElection
         /// @param llu The last log update seen by the master.
         /// @param content The topic content.
         /// @throws ObserverInconsistencyException Raised if an
-        /// inconsisency was detected.
+        /// inconsistency was detected.
+        ["swift:identifier:`init`"]
```

Comment:
I would remap it to initialize; I find `init` very confusing.

Reply:
Sure, changed to `initialize`.